.github/workflows/create-release.yml

```diff
@@ -25,7 +25,8 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 16
-      - run: npm ci
+        # There is no `package-lock.json` file
+      - run: npm install
```

Comment:
if the package json has deps, we should add the package-lock onto the root

Reply:
Sounds like something that probably should have a SOP.